exporter/dynatraceexporter/README.md

```diff
@@ -24,6 +24,11 @@ The Dynatrace exporter is enabled by adding a `dynatrace` entry to the `exporter
 All configurations are optional, but if an `endpoint` other than the OneAgent metric ingestion endpoint is specified then an `api_token` is required.
 To see all available options, see [Advanced Configuration](#advanced-configuration) below.
 
+> When using this exporter, it is RECOMMENDED to configure your SDKs to export DELTA metrics. 
+> When receiving CUMULATIVE metrics, this exporter will perform CUMULATIVE to DELTA conversions. 
```

Comment:
```suggestion
> When receiving CUMULATIVE metrics, this exporter will perform CUMULATIVE to DELTA conversion. 
```

Reply:
Applied in [25bd4c1](https://github.com/dynatrace-oss-contrib/opentelemetry-collector-contrib/pull/2009/commits/25bd4c123a1d57fb9e5fcb11ed0c7545e419be4a). Thanks. :slightly_smiling_face: